client/orderbook/orderbook_test.go

```diff
@@ -505,33 +530,31 @@ func TestOrderBookUnbook(t *testing.T) {
 			),
 			wantErr: false,
 		},
-		// May want to re-implement strict sequence checking. Might use these tests
-		// again.
-		// {
-		// 	label: "Unbook sell order with outdated sequence value",
-		// 	orderBook: makeOrderBook(
```

Comment:
Hmm, I see [relevant comment here](https://github.com/decred/dcrdex/pull/278#discussion_r415904036) from long time ago, but it doesn't quite answer why these were commented out in the first place ...